requirements/fabric/base.txt

```diff
@@ -6,4 +6,4 @@ torch>=1.10.0, <=1.13.1
 fsspec[http]>2021.06.0, <2022.6.0
 packaging>=17.1, <=21.3
 typing-extensions>=4.0.0, <=4.4.0
-lightning-utilities>=0.4.2, <0.6.0
+lightning-utilities>=0.4.2, <0.7.0
```

Comment:
You have to increase the min version here, as you are using code that's only in the newest

```suggestion
lightning-utilities>=0.7.0, <0.8.0
```

Same in the pytorch reqs

Reply:
So the latest version is 0.6.0 https://pypi.org/project/lightning-utilities/
But bumping the min version makes sense. Thanks